support-frontend/assets/helpers/productPrice/weeklyProductPrice.js

```diff
@@ -22,28 +33,38 @@ const getPromotionPrice = (country: IsoCountry, period: WeeklyBillingPeriod, pro
   ].join('');
 };
 
+function getPromotion(
+  productPrices: ProductPrices,
+  country: IsoCountry,
+  billingPeriod: BillingPeriod,
+  fulfilmentOption: ?FulfilmentOptions,
+): ?Promotion {
+  return genericGetPromotion(
+    productPrices,
+    country,
+    billingPeriod,
+    fulfilmentOption,
+    NoProductOptions,
+  );
+}
+
+function regularPrice(
```

Comment:
And here? Is this a pattern I'm missing?

Reply:
Also, should it be something like `getRegularPrice` and `genericGetRegularPrice`?

Reply:
Same as above